packages/flutter_tools/lib/src/ios/ios_emulators.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import '../base/file_system.dart';
+import '../base/platform.dart';
+import '../base/process.dart';
+import '../emulator.dart';
+import '../globals.dart';
+import 'ios_workflow.dart';
+
+// TODO(dantup): Is there a better name for this? We already have IOSSimulator classes
+// that represent *running* simulators, but this is about "unlaunched images"...
+class IOSEmulators extends EmulatorDiscovery {
```

Comment:
IOSSimulatorDiscovery?

Reply:
I think using the word Simulators is confusing, as we already use that for *running* devices (eg. we have a `class IOSSimulators extends PollingDeviceDiscovery`.

For Android we have "devices" (running) and "emulators" (AVDs), but for iOS we already had "devices" (running hardware) and "simulators" (running simulators). I think keeping it as "emulators" probably is least confusing for now; but probably it'd have been better if the existing "simulators" was named something less ambiguous (since we can have both offline and online simulators, whereas for devices we call the offline ones emulators - though that's still also kinda confusing).

I think probably I should just remove the TODO for now since I don't have any better idea (the more correct names are more ambiguous/confusing).

Reply:
We may need to differentiate between a running simulator, and a class which can start simulators. In any case, it's not a major issue; keeping the name as is and removing the TODO sounds good.